queries.sql

```diff
@@ -22,4 +22,36 @@ SELECT * FROM ANIMALS WHERE NEUTERED IS TRUE;
 SELECT * FROM ANIMALS WHERE NAME NOT LIKE 'Gabumon';
 
 -- Find all animals with a weight between 10.4kg and 17.3kg (including the animals with the weights that equals precisely 10.4kg or 17.3kg)
-SELECT * FROM ANIMALS WHERE WEIGHT_KG >= 10.4 AND WEIGHT_KG <= 17.3;
\ No newline at end of file
+SELECT * FROM ANIMALS WHERE WEIGHT_KG >= 10.4 AND WEIGHT_KG <= 17.3;
+
```

Comment:
- [ ] I think you forget to add this query here to set all the  species column to unspecified so kindly add it since it is one of the requirements ✅♻✨💯

![image](https://user-images.githubusercontent.com/58553711/208214314-5da1bed3-2037-410d-8c39-bfb1f4c178e9.png)

Reply:
Actually, I didn't leave it out. If you check the screenshots you'll notice it is there

Reply:
- I checked but I mean you didn't add it in the file ✅